core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -801,67 +820,76 @@ public void run()
       System.exit(0);
     }
 
-    try
-    {
-      workload.init(props);
-    }
-    catch (WorkloadException e)
-    {
-      e.printStackTrace();
-      e.printStackTrace(System.out);
-      System.exit(0);
-    }
+    final Tracer tracer = new Tracer.Builder("YCSB " + workload.getClass().getSimpleName()).conf(conf).build();
+
+    try (final TraceScope span = tracer.newScope("Client#workload_init")) {
+
+      try
+      {
+        workload.init(props);
+      }
+      catch (WorkloadException e)
+      {
+        e.printStackTrace();
+        e.printStackTrace(System.out);
```

Comment:
Is there a need to flush the "tracer" at here before System.exit(0)? The reason I ask is I'm not sure how the automatically scoped TraceScope works with abnormal exit like System.exit(), i.e., whether span.finalize() would be called in the event of System.ext()... (My understanding is the process is terminating, there is no need to do GC and reclaim out of scope variables anymore, so the flush won't happen...).

Or we can move System.exit(0) outside the scope of TraceScope.